src/Components/Navigation.tsx

```diff
@@ -0,0 +1,20 @@
+import { NavLink } from 'react-router-dom';
+
+export const Navigation = () => (
+  <nav className="is-flex mb-4">
+    <NavLink
+      to="/"
+      className={({ isActive }) => (
+        (isActive && 'button is-info') || 'button is-white')}
```

Comment:
now – great, but you could move it to a separate function so that you don't duplicate yourself here and with the following `NavLink`
```suggestion
      className={({ isActive }) => (
        (isActive && 'button is-info') || 'button is-white')
      }
```